client/src/pages/ChefProfile.js

```diff
@@ -1,15 +1,93 @@
-/**Comment added to avoid merge conflict with ongoing development  */
-// import React, { useContext } from "react";
-// import { AuthContext } from "../context/auth-context";
+import React, { useState, useEffect, useContext } from "react";
+import { Grid, Hidden, Box, Typography, Paper } from "@material-ui/core";
+import ChefSideBar from "../components/ChefProfile/ChefSideBar";
+import ChefRecipes from "../components/ChefProfile/ChefRecipes";
+import { makeStyles } from "@material-ui/core/styles";
+import { RecipeContext, RecipeDispatchContext } from "../context/RecipeContext";
+import { UserContext } from "../context/UserContext";
+import axios from "axios";
+import { getRecipesByChef } from "../actions/recipeActions";
+
+const ChefProfile = ({ history }) => {
+  const useStyles = makeStyles((theme) => ({
+    chefMenuName: {
+      marginTop: theme.spacing(10),
+      marginBottom: theme.spacing(6),
+    },
+  }));
+
+  const classes = useStyles();
+
+  const { recipes } = useContext(RecipeContext);
+  const dispatch = useContext(RecipeDispatchContext);
+
+  const { userInfo } = useContext(UserContext);
+  const [profile, setProfile] = useState(null);
+
+  useEffect(() => {
+    if (userInfo && userInfo.isChef) {
+      const getProfileAndRecipes = async () => {
+        const res = await axios.get(`/api/chefProfiles/${userInfo._id}`);
+        if (res.data) {
+          setProfile(res.data.chefProfile);
+          getRecipesByChef(dispatch, res.data.chefProfile._id);
+        }
+      };
+      getProfileAndRecipes();
+    } else if (!userInfo) {
+      history.replace("/login");
+    }
+  }, [dispatch, userInfo, history]);
+
+  const chefInfosAndRecipes = {
```

Comment:
Hey @jessdesr , The thing is that there was no ticket for integrating the ui to backend for this component , I just tried to make it somewhat integrated for the purpose of presentation. So it's not a done job. there are some fields in sideBar of the page that will check if the data is there and if not will use the dummy data. Also We still need to take care of location field, with reverse geocoding or having an address field. So that we can use the field in this component, for now it is just showing this dummy data